test/helper.py

```diff
@@ -4,12 +4,10 @@
 
 from mathics.session import MathicsSession
 
-session = MathicsSession(add_builtin=True, catch_interrupt=False)
-
-# Set up a Mathics3 session with definitions.
+# Set up a Mathics session with definitions.
 # For consistency set the character encoding ASCII which is
 # the lowest common denominator available on all systems.
-session = MathicsSession(character_encoding="ASCII")
```

Comment:
Now we create the builtin symbols  once, so the parameter in this second call does not have any effect.